app/controllers/announcements_controller.rb

```diff
@@ -15,10 +15,8 @@ def new
 
   def create
     @announcement = Announcement.new(content_params)
-    if @announcement.save!
+    if @announcement.save! 
       redirect_to announcements_path
-    else
```

Comment:
this part looked good. why get rid of it?

Reply:
Refactored later